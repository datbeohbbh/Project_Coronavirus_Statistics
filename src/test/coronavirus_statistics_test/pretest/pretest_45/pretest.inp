50 100
q 1 6 484
? 4 1
q 1 29 307
q 1 16 175
q 1 5 131
? 7
q 2 5 5
? 4 1
q 1 38 450
q 1 34 222
? 4 1
? 4 6
q 1 40 451
? 7
? 4 6
q 1 37 68
q 1 12 193
? 6
q 1 23 51
q 1 37 487
q 1 8 337
? 4 10
q 1 4 386
q 1 47 479
? 4 2
? 7
? 4 4
q 1 44 92
? 4 10
q 3 8 285
q 3 8 49
q 2 12 166
? 7
? 4 10
? 4 5
q 1 17 126
? 4 10
q 1 24 269
q 1 5 310
? 6
? 7
q 1 22 114
? 4 12
q 3 37 96
q 1 48 295
q 2 29 114
q 1 48 161
q 1 2 230
q 3 38 119
? 7
q 3 17 80
q 2 22 64
? 4 14
q 1 21 201
q 1 27 343
? 7
? 4 8
? 4 7
? 4 7
? 4 1
? 4 2
q 2 17 21
? 7
q 1 31 34
q 2 31 24
? 4 4
? 7
q 3 4 69
q 3 4 140
? 4 15
q 2 5 62
q 3 44 87
? 4 2
q 2 22 25
q 1 3 116
? 4 1
q 3 47 324
q 3 8 1
? 7
? 6
q 3 22 16
q 1 35 422
? 4 8
? 4 17
? 4 13
? 6
? 4 22
? 6
? 7
? 4 13
? 6
? 4 10
? 7
? 4 14
? 7
? 7
? 6
? 4 20
? 7
? 4 14