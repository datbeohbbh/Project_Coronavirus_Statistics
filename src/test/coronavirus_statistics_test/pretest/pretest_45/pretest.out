Query ? 4 1 :
The answer is the city with the index 6 
- Information of the city 6 -
-> Number of current patients : 484
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 1 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 126
-> Has recovered : 5
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 126
-> Has recovered : 5
-> Was dead : 0

Query ? 4 6 :
The answer is the city with the index 6 
- Information of the city 6 -
-> Number of current patients : 484
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 6 :
The answer is the city with the index 40 
- Information of the city 40 -
-> Number of current patients : 451
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 5
Quantity: 5 people

Query ? 4 10 :
The answer is the city with the index 6 
- Information of the city 6 -
-> Number of current patients : 484
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 126
-> Has recovered : 5
-> Was dead : 0

Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 4 :
The answer is the city with the index 12 
- Information of the city 12 -
-> Number of current patients : 193
-> Has recovered : 0
-> Was dead : 0

Query ? 4 10 :
The answer is the city with the index 38 
- Information of the city 38 -
-> Number of current patients : 450
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 8
Quantity: 334 people

Query ? 4 10 :
The answer is the city with the index 38 
- Information of the city 38 -
-> Number of current patients : 450
-> Has recovered : 0
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 126
-> Has recovered : 5
-> Was dead : 0

Query ? 4 10 :
The answer is the city with the index 4 
- Information of the city 4 -
-> Number of current patients : 386
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 12
Quantity: 166 people

Query ? 7 :
The city with the highest death is city with the index 8
Quantity: 334 people

Query ? 4 12 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 436
-> Has recovered : 5
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 8
Quantity: 334 people

Query ? 4 14 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 436
-> Has recovered : 5
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 8
Quantity: 334 people

Query ? 4 8 :
The answer is the city with the index 29 
- Information of the city 29 -
-> Number of current patients : 193
-> Has recovered : 114
-> Was dead : 0

Query ? 4 7 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 175
-> Has recovered : 0
-> Was dead : 0

Query ? 4 7 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 175
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 8 
- Information of the city 8 -
-> Number of current patients : 3
-> Has recovered : 0
-> Was dead : 334

Query ? 4 2 :
The answer is the city with the index 12 
- Information of the city 12 -
-> Number of current patients : 27
-> Has recovered : 166
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 8
Quantity: 334 people

Query ? 4 4 :
The answer is the city with the index 12 
- Information of the city 12 -
-> Number of current patients : 27
-> Has recovered : 166
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 8
Quantity: 334 people

Query ? 4 15 :
The answer is the city with the index 38 
- Information of the city 38 -
-> Number of current patients : 331
-> Has recovered : 0
-> Was dead : 119

Query ? 4 2 :
The answer is the city with the index 44 
- Information of the city 44 -
-> Number of current patients : 5
-> Has recovered : 0
-> Was dead : 87

Query ? 4 1 :
The answer is the city with the index 8 
- Information of the city 8 -
-> Number of current patients : 3
-> Has recovered : 0
-> Was dead : 334

Query ? 7 :
The city with the highest death is city with the index 8
Quantity: 335 people

Query ? 6 :
The city with the highest recovery is city with the index 12
Quantity: 166 people

Query ? 4 8 :
The answer is the city with the index 3 
- Information of the city 3 -
-> Number of current patients : 116
-> Has recovered : 0
-> Was dead : 0

Query ? 4 17 :
The answer is the city with the index 38 
- Information of the city 38 -
-> Number of current patients : 331
-> Has recovered : 0
-> Was dead : 119

Query ? 4 13 :
The answer is the city with the index 21 
- Information of the city 21 -
-> Number of current patients : 201
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 12
Quantity: 166 people

Query ? 4 22 :
The answer is the city with the index 48 
- Information of the city 48 -
-> Number of current patients : 456
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 12
Quantity: 166 people

Query ? 7 :
The city with the highest death is city with the index 8
Quantity: 335 people

Query ? 4 13 :
The answer is the city with the index 21 
- Information of the city 21 -
-> Number of current patients : 201
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 12
Quantity: 166 people

Query ? 4 10 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 175
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 8
Quantity: 335 people

Query ? 4 14 :
The answer is the city with the index 34 
- Information of the city 34 -
-> Number of current patients : 222
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 8
Quantity: 335 people

Query ? 7 :
The city with the highest death is city with the index 8
Quantity: 335 people

Query ? 6 :
The city with the highest recovery is city with the index 12
Quantity: 166 people

Query ? 4 20 :
The answer is the city with the index 35 
- Information of the city 35 -
-> Number of current patients : 422
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 8
Quantity: 335 people

Query ? 4 14 :
The answer is the city with the index 34 
- Information of the city 34 -
-> Number of current patients : 222
-> Has recovered : 0
-> Was dead : 0

